### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by jacoor**
Roadmap update. Contains products location change.

### CODE REVIEW

File: Dshop/apps/products_catalogue/models.py [hunk 1]
```diff
@@ -117,20 +131,24 @@ def get_attributes(self):
...
         return self.availability
 
 
 class ProductImage(models.Model):
-    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name='images')
-    image = models.ImageField(upload_to='media/')
+    product = models.ForeignKey(
+        Product, on_delete=models.CASCADE, related_name='images'
+    )
+    image = models.ImageField(upload_to='products/')
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/models.py]: Here is the change, rest is formatting.

**Comment by yanazPL** [comment]
I don't know if I'm right, but only changes from roadmap.md and .gitignore aren't already pushed.

In situation like that should you : 
update branch?
discard changes other than roadmap.md and .gitignore
just merge?

**Comment by michalkrzem** [comment]
The same. We have here more than only roadmap changes. Of course everything is clear, but i dont know if in this pull request.

**Comment by jacoor** [comment]
Yes and no :) after updating from dev it's only two files :-) 
Also guys, when you do review, there is option to request changes, no? So please use it.

**Review by yanazPL — APPROVED** [review]
Everything fine now

**Pull request merged**